src/explore-education-statistics-admin/src/components/editable/EditableContentBlock.tsx

```diff
@@ -116,7 +120,7 @@ const EditableContentBlock = ({
           <div
             // eslint-disable-next-line react/no-danger
             dangerouslySetInnerHTML={{
-              __html: content || '<p>This section is empty</p>',
+              __html: cleanContent || '<p>This section is empty</p>',
```

Comment:
I think we could create a new common component called `SanitizedHtml` which just accepts a `content` prop. We should then `useMemo` for its content.

We can re-use this for the above editable state (as a child of the `<div role="button">` element).